torchinductor/lowering.py

```diff
@@ -786,8 +786,9 @@ def arange(
     )
 
 
-@register_lowering(torch.linspace)
-def linspace(start, end, steps, *, dtype=None, device=None):
+@register_lowering([torch.linspace, aten.linspace])
```

Comment:
When are you seeing `torch.foo` operators btw?

Reply:
python_key_normalize captures `torch.*` tensor factories.